pkg/test/test.go

```diff
@@ -5,23 +5,30 @@ import (
 	"github.com/ory/dockertest"
 	"log"
 	"sync"
+	"time"
 )
 
 var err error
 var wait sync.WaitGroup
 var dockerPool *dockertest.Pool
 var dockerResources []*dockertest.Resource
+var configs []*ContainerConfig
 var cfgFilename = "config.test.yml"
 
 func init() {
 	dockerPool, err = dockertest.NewPool("")
+	dockerPool.MaxWait = 2 * time.Minute
 	dockerResources = make([]*dockertest.Resource, 0)
 
 	if err != nil {
 		log.Fatalf("Could not connect to docker: %s", err)
 	}
 }
 
+func Pool() *dockertest.Pool {
```

Comment:
this is a dead test function that can be removed

Reply:
✅